### PR TITLE
Retry pause min v2

### PR DESCRIPTION
**Pull request by r2evans**
reattempt at PR based on mis-rebased https://github.com/r-lib/httr/pull/383

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # httr 1.2.1.9000
 
+* Added `pause_min` to `RETRY` requests, allowing for sub-second
```
Comment (hadley) [inline NEWS.md]: Can you please add your github username like the other bullets?

Comment (r2evans) [inline NEWS.md]: Certainly, sorry I missed that. Since I have no "issue" number, would you prefer just `(@r2evans)` or `(@r2evans !461)`?

Comment (hadley) [inline NEWS.md]: Just your username please.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**